wear/src/main/java/io/homeassistant/companion/android/views/ChooseEntityView.kt

```diff
@@ -150,3 +155,46 @@ private fun ChooseEntityChip(
         colors = getFilledTonalButtonColors()
     )
 }
+
+@Preview(showSystemUi = true)
```

Comment:
Just to match the other Wear device screens can we set the device to large round? The screenshot in the description also shows a wearable screen against a phone UI :)

Reply:
Done!